### DISCUSSION TITLE
Status and charge-level

### DISCUSSION THREAD

**Discussion by ejepsen** [tobiasehlert/teslamateapi#241] [General]
Two simple questions:
1) /api/v1/cars/1/status returns {"error":"no info on this car ID"} but there is data in TeslaMate - what can be wrong?

2) I'm mainly looking to get the current charge-level of my Tesla but it's not clear how to retrieve this? Any guidance?

Thanks

**Comment by tobiasehlert** [comment]
@ejepsen, hope this helps:

1. Please check the version you are running from [TeslaMate](https://github.com/adriankumpf/teslamate) since there has been a change in vehicle information retrieval.
There is also a similar issue about this: #224

1. You can find the current charge level on the `/api/v1/cars/X/status` endpoint.
Inside `data.status.battery_details` where you have both `battery_level` and `usable_battery_level`.

Kind regards,
Tobias
